src/components/tab/style.css

```
@@ -0,0 +1,20 @@
+:host:not([open]) > label{
```

```suggestion
:host:not([open]) > label {
```
Add space after each selector to follow convetion. Otherwise, well done.